javascript/react/src/components/Codebook/CodebookFieldEditor/CodebookFieldEditor.stories.tsx

```diff
@@ -108,13 +58,77 @@ const codebookResult = {
 	],
 } as CodebookSchema
 
+const metadata: TableMetadata = {
```

Comment:
Do we have stories where the metadata is not explicitly defined like this, but is computed dynamically? It seems like we should support those cases. It would also be nice at some point if we could detect if the cached stats are invalid.

Reply:
At one point we stopped computing dynamic metadata, because it would be recomputed every time a component got remounted (i.e., on nav), so we moved to expecting the app to provide it and manage that state independently